region/wreckedship/main/Wrecked Ship Main Shaft.json

```
@@ -2486,6 +2519,19 @@
       ],
       "clearsObstacles": ["C"],
       "flashSuitChecked": true
+    },
+    {
+      "link": [8, 8],
+      "name": "Gain Flash Suit (Spikesuit)",
+      "requires": [
+        "f_DefeatedPhantoon",
+        {"canShineCharge": {"usedTiles": 17, "steepDownTiles": 5, "openEnd": 0}},
```

```suggestion
        {"canShineCharge": {"usedTiles": 20, "steepDownTiles": 5, "openEnd": 0}},
```
We called this 20 tiles in `id:100` to account for the conveyor